runconfig/config.go

```diff
@@ -51,7 +51,7 @@ func DecodeContainerConfig(src io.Reader) (*container.Config, *container.HostCon
 	return w.Config, hc, w.NetworkingConfig, nil
 }
 
-// validateVolumesAndBindSettings validates each of the volumes and bind settings
+// validateVolumesAndBindSettings validates each of the volumes and binds settings
```

Comment:
I think this one was correct before, because `bind` is being used as a noun, not a verb. So `bind settings` is the plural of `bind setting`.